src/lib/address/address.ts

```diff
@@ -28,6 +28,9 @@ const CHAIN_ID_TO_URN_SCHEME: Record<ChainId, string> = {
   [dogeChainId]: 'doge',
   [ltcChainId]: 'litecoin',
 }
+
+const DANGEROUS_ETH_URL_ERROR = 'modals.send.errors.qrDangerousEthUrl'
```

Comment:
Since we don't have access to i18n context here - if this looks too weird and we want to keep the console errors human-readable, we can keep this more human/dev friendly, and do the conversion to error message to translation path in `<QrCodeScanner />`